### PR TITLE
Updated README.md

### LINKED ISSUE #2: Add get started steps in readme
Update the [README](https://github.com/reactplay/docs/blob/main/README.md) with getting started with the project steps.

### PR DESCRIPTION
**Pull request by Misterr-H**
Fix of the #2 issue
Closes #2

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@Misterr-H , Can you link the related issue to this PR?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
 # docsite
 Source Code of the ReactPlay Documentation Website
+
+# Getting Started
+To get started, you need to have Node.js installed on your machine. You 
+can download it from [here](https://nodejs.org/en/download/).
+1. Clone the repository.
```
Comment (Sachin-chaurasiya) [inline README.md:7]: Can you add the command for cloning the repository?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
...
+```bash
+npm install
+```
+3. Start the development server by running the following command in the 
+root directory of the project:
+```bash
+npm run start
+```
+4. Open http://localhost:3000/ in your browser to view the website.
+5. Make changes to the website and see the changes reflected in the browser.
```
Comment (Sachin-chaurasiya) [inline README.md:19]:
```suggestion
5. Make changes to the code and see the changes reflected in the browser.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,28 @@
...
+directory of the project:
+```bash
+npm install
+```
+3. Start the development server by running the following command in the 
+root directory of the project:
+```bash
+npm run start
+```
+4. Open http://localhost:3000/ in your browser to view the website.
```
Comment (Sachin-chaurasiya) [inline README.md:18]:
```suggestion
4. Visit http://localhost:3000/ in your browser to view the website.
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,28 @@
...
+5. Make changes to the website and see the changes reflected in the browser.
+6. When you are done, stop the development server by pressing Ctrl+C in the terminal.
```
Comment (Sachin-chaurasiya) [inline README.md:20]:
```suggestion
6. Stop the development server by pressing Ctrl+C in the terminal.
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

**Comment by Misterr-H** [comment]
@Sachin-chaurasiya #2

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya #2 

@Misterr-H , Yeah can you link this issue to your PR?

**Comment by Misterr-H** [comment]
@Sachin-chaurasiya , umm sorry but how to do that?

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by atapas — APPROVED** [review]

**Pull request merged**